config/default.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/jezek/xgb"
+	"github.com/jezek/xgb/randr"
+	"github.com/jezek/xgb/xproto"
+)
+
+var (
+	AsoSxSViewerDir    = filepath.Join(os.Getenv("HOME"), ".aso_sxs_viewer")
+	AsoSxSViewerConfig = filepath.Join(AsoSxSViewerDir, "config.textproto")
+	DefaultCSSSelector = struct {
```

Comment:
Why not DefaultCSSSelector = proto.CSSSelector{Selector: "input",Position: 7}

Reply:
Because the selector is of type string, so I needed to just save a string. I could use independent strings but I instead just combined them in this unnamed struct